tests/installation/bootloader_s390.pm

```diff
@@ -86,7 +86,7 @@ sub prepare_parmfile {
 
     # we have to hardcode the hostname here - the true hostname would
     # create a too long parameter ;(
-    my $instsrc = get_var('INSTALL_SOURCE', 'ftp') . '://' . get_var('REPO_HOST', 'openqa') . '/';
+    my $instsrc = get_var('INSTALL_SOURCE', 'ftp') . '://ftp:ftp@' . get_var('REPO_HOST', 'openqa') . '/';
```

Comment:
```suggestion
    my $repo_host = get_var('REPO_HOST', 'openqa');
    if (check_var('INSTALL_SOURCE', 'ftp')) {
      $repo_host = 'ftp:ftp@' . $repo_host;
    }
    my $instsrc = get_var('INSTALL_SOURCE', 'ftp') .  $repo_host . '/';
```

Reply:
I agree with this suggested change which properly handles an FTP install source that requires a user and password.

Reply:
> 

+1